which-key.el

```diff
@@ -40,6 +40,7 @@
 
 (require 'cl-lib)
 (require 'button)
+(require 'god-mode nil t)
```

Comment:
This doesn't seem necessary and could be undesirable for some users. If this is for the compiler we should just declare the functions and variables.